appservice/src/createAppService/AppInsightsListStep.ts

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApplicationInsightsManagementClient } from "azure-arm-appinsights";
+import { ApplicationInsightsComponent, ApplicationInsightsComponentListResult } from "azure-arm-appinsights/lib/models";
+import { isString } from "util";
+import { AzureWizardPromptStep, createAzureClient, IAzureNamingRules, IAzureQuickPickItem, IAzureQuickPickOptions, IWizardOptions } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+import { localize } from "../localize";
+import { AppInsightsCreateStep } from "./AppInsightsCreateStep";
+import { AppInsightsLocationStep } from "./AppInsightsLocationStep";
+import { AppInsightsNameStep } from "./AppInsightsNameStep";
+import { IAppServiceWizardContext } from "./IAppServiceWizardContext";
+
+export const appInsightsNamingRules: IAzureNamingRules = {
+    minLength: 1,
+    maxLength: 255,
+    invalidCharsRegExp: /[^a-zA-Z0-9\.\_\-\(\)]/
+};
+
+const skipForNow: string = '$(circle-slash) Skip for now';
```

Comment:
In other places I use `$(clock)`
![Screen Shot 2019-08-13 at 9 05 33 AM](https://user-images.githubusercontent.com/11282622/62957436-a234c200-bda9-11e9-93d6-88e3946631c1.png)

I'd prefer to stay consistent unless you have a strong objection